workloads/run_etcd_tests_fromgit.sh

```diff
@@ -181,28 +181,41 @@ EOF
 oc adm policy add-scc-to-user -n my-ripsaw privileged -z benchmark-operator
 oc adm policy add-scc-to-user -n my-ripsaw privileged -z backpack-view
 
-cat << EOF | oc create -n my-ripsaw -f -
+oc apply -n my-ripsaw -f - <<< "
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
   name: fio-benchmark
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    server: $_es
-    port: $_es_port
-  clustername: $cloud_name
+    server: ${_es}
+    port: ${_es_port}
+  fio_path: /var/tmp
+  clustername: ${cloud_name}
   test_user: ${cloud_name}-ci
   metadata_collection: true
   metadata_sa: backpack-view
   metadata_privileged: true
   workload:
-    name: byowl
+    name: fio_distributed
```

Comment:
I previously had the fio_distributed work here, however it did not work :/

I saw if we bump our FIO version it might be fixed?